site/content/faq/300-typescript.md

```diff
@@ -0,0 +1,8 @@
+---
+question: How do I use TypeScript with Sapper?
+---
+
+The default template is currently setup only with JavaScript support, but here are some templates that community members have created with TypeScript support:
+
+* https://github.com/babichjacob/sapper-typescript-graphql-template
```

Comment:
I would probably point to the integrations section of the sveltejs/integrations repository rather than partially duplicating the content of it here.

Reply:
There aren't listed on `sveltejs/integrations` and I'm not sure there's a really good place to put them there since they're specific to Sapper and not Svelte in general

Reply:
Sub-section "Sapper templates" under "Templates (boilerplates)"

Reply:
Yeah I suppose that's reasonable. Ok then this list seems fine to stay here.